packages/new-polymath-ui/package.json

```diff
@@ -57,7 +59,8 @@
     "docz": "^0.12.13",
     "docz-plugin-css": "^0.11.0",
     "docz-theme-default": "^0.12.13",
-    "jest": "^23.6.0",
+    "jest": "^24.0.0-alpha.9",
+    "jest-dom": "^3.0.0",
```

Comment:
With recent release of React 16.8, you should update these to latest version.

Reply:
Just realised it's more complicated than that. Jest 24 comes with TS support and apparently we can remove `babe-core bridge` entirely? https://jestjs.io/blog/2019/01/25/jest-24-refreshing-polished-typescript-friendly#typescript-support

Reply:
So I tried to update and it works fine actually (I didn't touch any Babel stuff cause I broke everything...).
However in this PR you added `jest/jest-dom` 2 times in dependencies and dev-dependencies!

Reply:
@Grsmto hey, babel-core bridge hopefully. But I dont think so, I think module resolver requires it, though we shoudl check.

I'll fix the duped dependencies btw, thanks!